mlx_engine/vision/vision_model_kit.py

```diff
@@ -38,6 +40,47 @@ def __init__(
         self.model_path = model_path
         self._initializer()
 
+    @staticmethod
+    def _fix_qwen2_5_vl_image_processor(model_path: Path):
+        """
+        Register Qwen2_5_VLImageProcessor as Qwen2VLImageProcessor in AutoImageProcessor
+        This is needed because Qwen2_5_VLImageProcessor was deleted from transformers
```

Comment:
```suggestion
        This is needed because Qwen2_5_VLImageProcessor was deleted from transformers, but legacy versions of the model used this
```
Or something that mentions why it matters that this processor was deleted from transformers